tools/crates/cli/src/commands/starknet/transfer_args.rs

```diff
@@ -1,3 +1,5 @@
+use std::str::FromStr;
```

Comment:
you can avoid importing this by using `args.channel_id.parse()`

Reply:
Done in b70f8769382dbbe20e72136e6dd95058270d64b6